sdl2-vga-terminal/src/exceptions/exceptions.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+
+#include <stdexcept>
+
+namespace exceptions
+{
+class LogicalRenderError : public std::runtime_error
+{
+public:
+    LogicalRenderError(const std::string& message) : runtime_error(message) {}
+};
+
+class SdlWasNotInit : public std::runtime_error
+{
+public:
+    SdlWasNotInit(const std::string& message) : runtime_error(message) {}
+};
+
+class WindowError : public std::runtime_error
+{
+public:
+    WindowError(const std::string& message) : runtime_error(message) {}
+};
+
+class RendererError : public std::runtime_error
+{
+public:
+    RendererError(const std::string& message) : runtime_error(message) {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'RendererError' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5472420)